internal/orchestrator/applications_handler.go

```diff
@@ -81,6 +84,27 @@ func (handler ApplicationsHandler) List(w http.ResponseWriter, _ *http.Request)
 	for _, rec := range records {
 		list.Applications = append(list.Applications, Application{ID: rec.ID, IntegrationId: rec.IntegrationId, ObjectId: rec.ObjectId, Name: rec.Name, Description: rec.Description, ProviderName: integrationNamesById[rec.IntegrationId], Service: rec.Service})
 	}
+
+	// sort by "Provider" so that all app resources from a platform are grouped together.
+	// sort ProviderName as the first (asc) order and Service as the second (asc) order.
+	sort.Slice(list.Applications, func(i, j int) bool {
+		providerComp := strings.Compare(list.Applications[i].ProviderName, list.Applications[j].ProviderName)
+		serviceComp := strings.Compare(list.Applications[i].Service, list.Applications[j].Service)
+		switch providerComp {
```

Comment:
@jingchu000 - Could be further simplified
```
switch providerComp {
    case 0: return serviceComp <= 0
   default: return providerComp < 0
}
```

Reply:
good, learn a lot

Reply:
Thanks